webapp/src/pages/Dashboard.tsx

```diff
@@ -85,6 +85,8 @@ const Dashboard = () => {
       {isPipelineSelected(pipeline) && (
         <PreviewCard
           title="Performance Analysis"
+          to={`/${jobId}/performance_analysis${searchString}`}
+          linkButtonText="Compare models"
```

Comment:
With `pipelines` instead of `models`?
```suggestion
          linkButtonText="Compare pipelines"
```